### PR TITLE
fix(perf): Fix early truncation in transaction search

### PR DESCRIPTION
**Pull request by 0Calories**
Slight CSS adjustment for the Transaction Search Bar, the transaction names were getting truncated too early.

### Before
image

### After
image

In a follow up PR, I will look to allow hovering over the items to view the full name, even if it is truncated (since the names will still be truncated if the screen doesn't have enough space for the name)

### CODE REVIEW

**Review by malwilley — COMMENTED** [review]
One small design suggestion: Might want to add a margin-left to the icon so the text doesn't run into it:

CleanShot 2022-12-21 at 11 09 19

**Comment by 0Calories** [comment]
> One small design suggestion: Might want to add a margin-left to the icon so the text doesn't run into it:
> 

@malwilley  Thanks for the suggestion! Instead of adding a margin-left (decided against doing this since I'd need to make a new wrapper component), I adjusted the width of the text from `100%` to `95%` which seems to achieve the desired effect:

image

**Review by malwilley — APPROVED** [review]
>  Instead of adding a margin-left (decided against doing this since I'd need to make a new wrapper component), I adjusted the width of the text from 100% to 95% which seems to achieve the desired effect

Having a set margin is slightly better because it's more consistent across different widths and allows us to modify spacing throughout the app if you use the `space()` utilities - but I digress, this looks good 👍

**Comment by 0Calories** [comment]
Actually you've changed my mind, I do like the idea of consistent spacing across widths more, so I brought the wrapper component back in 😄

File: static/app/components/smartSearchBar/searchDropdown.tsx [hunk 1]
```diff
@@ -629,3 +631,7 @@ const Value = styled('span')<{hasDocs?: boolean}>`
 
   ${p => p.theme.overflowEllipsis};
 `;
+
+const IconOpenWrapper = styled('div')`
```
Comment (malwilley) [inline static/app/components/smartSearchBar/searchDropdown.tsx:635]: You should be able to do `styled(IconOpen)` here and not need an extra div

Comment (0Calories) [reply static/app/components/smartSearchBar/searchDropdown.tsx]: That's a better idea, updated!

**Review by malwilley — APPROVED** [review]

**Pull request merged**